README.md

```diff
@@ -118,45 +160,99 @@ This library has a `peerDependencies` listing for `react-dom`.
 You may also be interested in installing `jest-dom` so you can use
 [the custom jest matchers](https://github.com/gnapse/jest-dom#readme)
 
-## Usage
+## Setup
+
+### Globals
+
+There are several options you can add to your global test config that simplify
+the setup and teardown of tests in individual files. For example, you can ensure
+[`cleanup`](#cleanup) is called after each test and import additional
+assertions.
+
+To do this with Jest, you can add the
+[`setupTestFrameworkScriptFile`](https://facebook.github.io/jest/docs/en/configuration.html#setuptestframeworkscriptfile-string)
+option to your Jest config. The setup file can be anywhere, but if you're using
+create-react-app, use the default `src/setupTests.js`.
 
 ```javascript
-// __tests__/fetch.js
-import React from 'react'
-import {render, fireEvent, cleanup, waitForElement} from 'react-testing-library'
-// this adds custom jest matchers from jest-dom
+// jest.config.js
+module.exports = {
+  setupTestFrameworkScriptFile: require.resolve('./jest.setup.js'),
+  // ... other options ...
+}
+```
+
+```javascript
+// jest.setup.js
```

Comment:
Maybe we should recommend putting this file in a directory called `test` to avoid overfilling the root directory?

Reply:
TBH I prefer putting that stuff in root initially and cleaning things up only if it gets overwhelming. Plus, `test` sounds like a place to put tests and not just a config directory.

Reply:
Added a line here: https://github.com/kentcdodds/react-testing-library/pull/166/files#diff-04c6e90faac2675aa89e2176d2eec7d8R179